test/integration/schema/index.js

```diff
@@ -1238,6 +1238,51 @@ module.exports = (knex) => {
               });
           });
         });
+
+        it('#2767 - .renameColumn should not drop the auto incremental', () => {
```

Comment:
see my comment. is it not possible to make test db-agnostic?

Reply:
the tests are very different depending on the db type. So I've added tests per db to validate it doesn't happen in other dbs and to proof that it is only a mysql problem